agent/api/task/task_test.go

```diff
@@ -479,61 +469,91 @@ func convertSCPort(port uint16) nat.Port {
 }
 
 // TestDockerHostConfigSCBridgeMode verifies port bindings and network mode overrides for each
-// container in an SC-enabled bridge mode task. The test task is consisted of the SC container, a regular container,
+// container in an SC-enabled bridge mode task with default/user-specified dynamic host port range.
+// The test task is consisted of the SC container, a regular container,
 // and two pause containers associated with each.
 func TestDockerHostConfigSCBridgeMode(t *testing.T) {
```

Comment:
This test `TestDockerHostConfigSCBridgeMode()` has been reviewed in https://github.com/aws/amazon-ecs-agent/pull/3585.